scripts/us_census/pep/pep_by_asrh/preprocess.py

```diff
@@ -0,0 +1,945 @@
+
+    Returns:
+        str: Age Bucket Value
+    """
+    if age_grp == 0:
+        return ""
+    if age_grp == 85:
+        return "85"
+    start = 5 * (age_grp - 1)
+    end = 5 * (age_grp) - 1
+    return f"{start}To{end}"
+
+
+def _get_age_grp2(age_grp: int) -> str:
+    """
+    Returns Age Groups using age_grp index as below
+    0: "0"
+    1: 1To4
+    2: 5To9
+    3: 10To14
+    ...
+    ...
+    85: 85To89
+    Args:
+        age_grp (int): Age Group Bucket Index
+
+    Returns:
+        str: Age Bucket Value
+    """
+    if age_grp == 0:
+        return "0"
+    if age_grp == 1:
+        return "1To4"
+    start = 5 * (age_grp - 1)
+    end = 5 * (age_grp) - 1
+    return f"{start}To{end}"
+
+
+def _add_measurement_method(df: pd.DataFrame, src_col: str,
+                            tgt_col: str) -> pd.DataFrame:
+    """Adds Measurement Method either CensusPEPSurvey or
+    dcAggregate/CensusPEPSurvey to tgt_col column.
+    Args:
+        df (pd.DataFrame): Input DataFrame
+        src_col (str): SV Column Name
+        tgt_col (str): Measurement Method Column
+
+    Returns:
+        pd.DataFrame: DataFrame with New Columns
+    """
+    df[tgt_col] = df[src_col].str.split("_").str[-1]
+    df[tgt_col] = df[tgt_col].str.replace(r"^(?!computed).*",
+                                          "dcs:CensusPEPSurvey",
+                                          regex=True)
+    df[tgt_col] = df[tgt_col].str.replace('computed',
+                                          "dcs:dcAggregate/CensusPEPSurvey",
+                                          regex=False)
+    df[src_col] = df[src_col].str.replace("_computed", "")
+    return df
+
+
+def _load_df(path: str,
+             file_format: str,
+             header: str = None,
+             skip_rows: int = None,
+             encoding: str = "UTF-8") -> pd.DataFrame:
+    """
+    Returns the DataFrame using input path and config
+    Args:
+        path (str): Input File Path
+        file_format (str): Input File Format
+        header (str, optional): Input File Header Index. Defaults to None.
+        skip_rows (int, optional): Skip Rows Value. Defaults to None.
+        encoding (str, optional): Input File Encoding. Defaults to None.
+
+    Returns:
+        df (pd.DataFrame): Dataframe of input file
+    """
+    df = None
+    if file_format.lower() == "csv":
+        df = pd.read_csv(path, header=header, encoding=encoding)
+    elif file_format.lower() == "txt":
+        df = pd.read_table(path,
+                           index_col=False,
+                           delim_whitespace=True,
+                           engine='python',
+                           header=header,
+                           skiprows=skip_rows,
+                           encoding=encoding)
+    elif file_format.lower() in ["xls", "xlsx"]:
+        df = pd.read_excel(path, header=header)
+    return df
+
+
+def _merge_df(df_1: pd.DataFrame, df_2: pd.DataFrame) -> pd.DataFrame:
+    if df_1 is None:
```

Comment:
Why are we assuming that it's ok for df_1 to be None and not df_2?
This seems to be a very specific use-case for merge (you are also merging on axis=0). _merge_df() is also called in just one place. I wonder if its better to just use pd.concat directly instead of having this separate _merge_df() function? But if you do use it, please provide slightly more helpful documentation.